Dockerfile

```diff
@@ -5,20 +5,18 @@ ENV CEREBRO_VERSION=0.6.6 \
 
 WORKDIR /opt/cerebro
 RUN set -ex;\
-  addgroup -S cerebro && adduser -S -G cerebro cerebro; \
+  adduser -S cerebro; \
   apk add --no-cache \
     bash \
-    ca-certificates \
+    curl \
```

Comment:
this makes no odds anyway since openjk8-jre ends up depending on ca-certificates anyway ...